src/en/news/blog/2022/improving-dashboard-accessibility-for-blind-and-visually-impaired-ceph-users/index.md

```diff
@@ -0,0 +1,134 @@
+
+#### Login Page
+
+By improving the color contrast of the login form, input fields users can now clearly read the labels and use the keyboard to navigate the inputs.
+
+![login page](images/login.png)
+
+#### Landing Page
+
+The Dashboard page displays key monitoring information about cluster status, capacity and performance. We improved on the following aspects:
+
+- updated the cluster labels (HEALTH_WARN, HEALTH_OK, HEALTH_ERROR) to be more descriptive (Warning, Ok, Error)
+
+- Used darker colors for labels to make them more legible
+- Added accessible names to informative icons for screen readers
+
+![Dashboard page](images/dashboard.png)
+
+#### Navigation
+
+The navigation component enables quick access to all the Dashboard's sections. I was able to make the following accessibility updates:
+
+- Add accessible roles for all dropdown menus for easier detection by screen readers
+- Used darker background colors and increased label font size for better legibility
+
+![Navigation](images/navigation.png)
+
+#### Datatables
+
+The datatable component is widely used in the Dashboard to display more detailed information. The improvements made here include:
+
+- Added accessible labels for row selection checkboxes for better identification by screen readers
+- Added a custom pagination component with better UX and accessibility
+- Changed table action button label colors from red to cyan for better legibility and focus
+
+![Datatable](images/datatable.png)
+
+Many more sub-components were also updated and can be found on [GitHub](https://github.com/ceph/ceph/pulls/nsedrickm).
+
+### Accessibility testing
+
+To prevent future regressions, we added end to end accessibility tests for high impact components like the health page and navigation. If any future regressions on these components occurs, a tabulated error description is generated and added to the Cypress log for debugging.
+
+![Error message](images/error.png)
+
+More information on accessibility e2e testing can be found in the dashboard developer [documentation](https://docs.ceph.com/en/latest/dev/developer_guide/dash-devel/#accessibility).
+
+## Challenges faced
+
+While working on this project, I ran into several challenges. Actually tracking down the code where fixes needed to be made was quite a task most times, given how large the Ceph codebase is. Using `grep` helped me find code sections a lot faster.
+
+Also, some components like the datatable are actually provided by a third party library. This made it a bit challenging to improve on the accessibility of certain parts. Fortunately, we could develop and use custom templates instead with better accessibility. In some extreme cases, whole components had to be replaced like the table pagination component.
+
+In addition, some of the updates I made altered the QA tests currently in place for the Dashboard. This earned me several days of debugging to figure out what I was doing wrong and how to seek help when stuck.
```

Comment:
This is really interesting! We should probably have a retrospective with all GSoC @ceph/2022-interns, to learn more about their issues getting used to Ceph. What do you think: @ljflores @alimaredia ?

Reply:
+1 @epuertat!